docs/partials/component-list.html

```diff
@@ -53,6 +53,7 @@
   <li class="docs-nav-title">Containers</li>
   <li{{#ifpage 'accordion'}} class="current"{{/ifpage}}><a href="accordion.html">Accordion <span class="label">JS</span></a></li>
   <li{{#ifpage 'callout'}} class="current"{{/ifpage}}><a href="callout.html">Callout</a></li>
+  <li{{#ifpage 'callout'}} class="current"{{/ifpage}}><a href="card.html">Card</a></li>
```

Comment:
Looks like this was copied from the previous line. Shouldn't `#ifpage 'callout'` should be `#ifpage 'card'`?